solutions/3645/01.py

```
@@ -1,27 +1,26 @@
 class Solution:
-    def maxTotal(self, value: List[int], limit: List[int]) -> int:
-        from collections import defaultdict
-        import heapq
-
-        # Group items by their limit values
-        groups = defaultdict(list)
-        for i, (v, l) in enumerate(zip(value, limit)):
-            groups[l].append(v)
+    def countPyramids(self, grid: List[List[int]]) -> int:
```

Ruff (0.14.10)

Undefined name `List`

(F821)